booking.sql

```diff
@@ -0,0 +1,41 @@
+BEGIN;
+
+DROP TABLE IF EXISTS authors;
+CREATE TABLE authors (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR NOT NULL,	
+	nationality VARCHAR,
```

Comment:
Take care with spaces. 
```suggestion
  id SERIAL PRIMARY KEY,
  name VARCHAR NOT NULL,	
  nationality VARCHAR,
```